src/pages/sok.tsx

```diff
@@ -0,0 +1,213 @@
+    >
+      <div className="py-[0px]">
+        <img src={image} className="md:h-[300px] lg:h-[300px] rounded-xl"></img>
+      </div>
+      <div
+        className={`${bgColor} ${borderColor} border-[3px] rounded-lg py-[25px] md:py-[25px] lg:py-[50px] px-[20px] bg-opacity-10 md:w-[300px] md:h-[300px] lg:w-[400px] lg:h-[300px]`}
+      >
+        <h2 className="text-center lg:text-3xl md: text-xl text-white">
+          {eventInfo[0]}
+        </h2>
+        <h3 className="text-center text-cerise  lg:text-xl md:mt-1 lg:mt-2"></h3>
+        <p className="text-white text-start mt-5 text-sm">{eventInfo[1]}</p>
+        <button className={`${textColor} text-start mt-3`} onClick={openModal}>
+          <u>{t.sok.extra}</u>
+        </button>
+
+        {/*MODAL*/}
+        {modalState && (
+          <div
+            className="fixed inset-0 flex items-center justify-center bg-black bg-opacity-60 backdrop-blur-lg"
+            ref={modalRef}
+            onClick={handleOverlayClick}
+          >
+            <div
+              className={`bg-white bg-opacity-70 w-[500px] max-h-[80vh] overflow-y-auto pb-5 flex flex-col rounded-3xl`}
+              style={{ scrollbarWidth: "none", msOverflowStyle: "none" }}
+            >
+              <div className="relative py-[0px] justify-center flex flex-row">
+                <img src={image} />
+
+                <button
+                  className="absolute top-5 right-3 w-[50px] h-[50px] flex items-center justify-center"
+                  onClick={closeModal}
+                >
+                  <div className="absolute h-[50px] w-[5px] bg-white rounded-md rotate-45"></div>
+                  <div className="absolute h-[50px] w-[5px] bg-white rounded-md -rotate-45"></div>
+                </button>
+              </div>
+              <div className="px-5 mt-5">
+                <h2 className="text-center text-3xl text-black">
+                  {eventInfo[0]}
+                </h2>
+                <h3 className="text-center text-cerise text-2xl mt-2"></h3>
+                <ul className="list-none mt-2 text-center">
+                  {roles.map((role, index) => (
+                    <li key={index} className={index !== 0 ? "mt-8" : ""}>
+                      {role.split("\n").map((line, i) => (
+                        <p
+                          key={i}
+                          className={
+                            i === 0 ? "ml-0 text-left text-cerise" : "ml-0"
+                          }
+                        >
+                          {line}
+                        </p>
+                      ))}
+                    </li>
+                  ))}
+                </ul>
+                <p className="text-black text-start mt-5">
+                  {/* Content here */}
+                </p>
+              </div>
+            </div>
+          </div>
+        )}
+      </div>
+    </div>
+  );
+}
+
+export default function Students() {
```

Comment:
This the page function name should be changed to "Sok" or something similar

Reply:
Fixed